lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -313,25 +324,43 @@ dictionary GetInfoResponse {
 };
 
 dictionary PrepareSendRequest {
-    string invoice;
+    string payment_destination;
+    u64? amount_sat;
+};
+
+[Enum]
+interface PaymentDestination {
+    BIP21(LiquidAddressData address);
+    Bolt11(string invoice);
```

Comment:
Since in Bip21 we have the parsed data (LiquidAddressData) it would be more consistent if the Bolt11 contains the LNInvoice isntead of the string.